examples/wordcount/example.py

```diff
@@ -33,8 +33,8 @@ def process(self, key, value):
         words = value.split()
         log.debug(f'words list ({words})')
         for word in words:
-            count = self.word_count_store.get(word, 0)
-            self.word_count_store[word] = count + 1
+            count = self.word_count_store.get(word, '0')
+            self.word_count_store[word] = str(int(count) + 1)
```

Comment:
Switched to just use strings for keys and values for now until serialisation is implemented properly. I'm not quite sure why the publishing of word counts worked at all.

Reply:
This should work. We just need to create an asymmetric Serde as we have String in and Integer out. I'll add this to the e.g. for #43